tests/testsuite/clean.rs

```diff
@@ -432,7 +432,7 @@ fn assert_all_clean(build_dir: &Path) {
     }) {
         let entry = entry.unwrap();
         let path = entry.path();
-        if let ".rustc_info.json" | ".cargo-lock" | "CACHEDIR.TAG" =
+        if let ".rustc_info.json" | ".cargo-lock" | "CACHEDIR.TAG" | ".rustdoc_fingerprint.json" =
```

Comment:
I'm not sure what this is here for.

Reply:
This is here because the cargo clean test needs to avoid checking this `json` . It has been addressed.